ServerCore/Pages/Resources/pd2021.2203/RulesPartial.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿<div class="container">
+    <h2>Rules for Intern Puzzleday 2021: 2022 Edition</h2>
+    <h3>How to play</h3>
+    <ol>
+        <li><strong>HAVE FUN!</strong> (And if you're not having fun, let us know. If your team is running out of steam, let us know.)</li>
+        <li>None of the puzzles require you to do anything illegal, immoral, or that would cause Microsoft security to be concerned.</li>
+        <li>Everyone should have access to a computer during the event.</li>
+        <li>Access to a printer is optional. You may find some puzzles easier to solve on paper, but all puzzles are designed to be solveable on your computer.</li>
+        <li>You may use your computer to communicate with puzzle control and help you solve puzzles. You may also use any printed reference material you care to bring with you. You are encouraged to familiarize yourself with a number of common <a href="./Encodings">encodings</a>.</li>
+        <li>You can use any reference source on the internet. You can use friends or strangers as reference sources but not to help you solve puzzles (e.g., you could call someone to ask them a trivia question but not send them a whole list of trivia questions for them to research). You may not send entire puzzles to friends for help.</li>
+        <li>Writing a computer program to solve a puzzle is allowed, but this may take longer than solving the puzzle in its intended manner.</li>
+        <li>
+            Hacking puzzles is not in the spirit of the event and is strictly prohibited. This includes:
+            <ul>
+                <li>hacking into any computer system to find answers</li>
+                <li>giving false information to another team</li>
+                <li>trying to purloin answers from another team, and</li>
+                <li>colluding with other teams</li>
+            </ul>
+        </li>
+        <li>In order for a puzzle to be marked as solved, the correct answer must be entered into the correct answer submission box on the website and be submitted before the event ends.</li>
+        <li>Each team must be reachable by email during the event. New puzzles and corrections to puzzles will be sent out via email. <b>Ensure that you are set up to receive email from msph@puzzlehunt.org.</b></li>
+    </ol>
+    <h3>Team registration</h3>
+    <p>Teams should be between 4 and 6 people. You may register your team with only 1 person and fill out the roster later.</p>
+    <ol>
+        <li>The event is designed for Microsoft interns. Each intern may invite one friend to join their team. This person must be registered as a member of your team and will count against your limit of six people.</li>
+        <li>Non-intern Microsoft employees are not permitted to participate in this specific event.</li>
```

Comment:
If you *do* allow FTEs then this line needs to be removed

Reply:
No, we're definitely keeping this to interns; the FTE mention above was my token "crap I missed this" when I copied the old files from NIPD.